manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
After some testing, I'm not confident that `database/0` is submitting metrics. The monitor reported `0` when I stopped `bbs` on `database/0` and left it running on `database/1`. But reports `no data` when I stopped it on `database/1` and left it running on `database/0`. Worth noting that `cfdot` still works.